src/components/Editor/index.js

```diff
@@ -344,7 +339,7 @@ class Editor extends PureComponent<Props, State> {
     }
 
     if (conditionalPanelLine) {
-      return closeConditionalPanel();
```

Comment:
Hmm, I'd like to check elsewhere but I believe in functions with multiple actions we destructure for brevity